lib/manager/batect/extract.ts

```diff
@@ -76,10 +78,40 @@ function extractBundleDependencies(config: BatectConfig): PackageDependency[] {
   return deps;
 }
 
-export function extractPackageFile(
+function extractReferencedConfigFiles(
+  config: BatectConfig,
+  fileName: string
+): string[] {
+  if (config.include === undefined) {
+    return [];
+  }
+
+  const dirName = upath.dirname(fileName);
+
+  const paths = [
+    ...config.include.filter(
+      (include): include is string => typeof include === 'string'
+    ),
+    ...config.include
+      .filter(
+        (include): include is BatectFileInclude =>
+          typeof include === 'object' && include.type === 'file'
```

Comment:
Please move this to a module function too. Maybe a good idea is to move it to a new file `./utils.ts` so we reduce file size for better maintainabillity.